tests/components/backup/test_websocket.py

```diff
@@ -40,22 +40,13 @@ async def test_remove(
     await hass.async_block_till_done()
 
     with patch(
-        "homeassistant.components.backup.websocket.BackupManager._backups",
-        {TEST_BACKUP.slug: TEST_BACKUP},
-    ), patch(
-        "homeassistant.components.backup.websocket.BackupManager._loaded",
-        True,
-    ), patch(
-        "pathlib.Path.unlink"
-    ), patch(
-        "pathlib.Path.exists", return_value=True
-    ):
+        "homeassistant.components.backup.websocket.BackupManager.remove_backup",
+    ), patch("pathlib.Path.unlink"), patch("pathlib.Path.exists", return_value=True):
```

Comment:
I think we can remove the pathlib patches as we're patching a higher level now.

Reply:
correct.